components/places/src/db/schema.rs

```diff
@@ -775,4 +775,75 @@ mod tests {
         drop(db);
         db_file.upgrade_to(16);
     }
+
+    #[test]
+    fn test_db_migration_sanity() {
```

Comment:
suggestion: Let's try to find a different term for this if possible, please—maybe `test_all_upgrades`, to match `autofill` and `webext-storage`?

I know it's a common phrase (and I've definitely used it before), but it can have hurtful connotations, and I wonder if there's a more precise term we can use to describe this test—and improve (rip)grep-ability while we're at it!

Reply:
Thanks for raising this!  I hadn't even considered the issue.  I just pushed a new commit that fixes the naming.